source/_posts/2017-07-07-you-may-not-be-a-polyglot-but-your-code-needs-to-be.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "You May Not Be a Polyglot, but Your Code Needs to Be"
+author: Denisse Osorio de Large
+date: 2017-07-07
+tags: [engineering, polyglot, localization, internationalization, grace, hopper]
+---
+
+Last October, I had the privilege to speak at Grace Hopper, the world's largest gathering of women technologists. It was impressive to see 15,000 fellow female technologists gather together to share their experiences and technical expertise. To look into the audience, and see a room full of other female engineers was a great experience! 
+
+My talk was titled, You May Not Be a Polyglot, but Your Code Needs to Be.
+One definition of the term polyglot is "composed in many languages." During my talk, I discussed common misconceptions about writing code for global audiences, and I provided tips to help position engineers for successful in these types of projects. 
+
+Having worked in engineering, translation, and localization for more than over a decade, I have seen first-hand how projects struggle through the globalization process. A properly globalized solution is generally achieved by ensuring the appropriate internationalization, localization, and translation steps have taken place; however, as developers, we have misconceptions that make this process more costly and less effective than it needs to be.
+
+To help set the stage, I began my talk by speaking in Spanish. I love watching the audience react to my comments, as I usually say something silly during the introduction, so those who understand react one way, while others in the audience tend to look puzzled.
+
+One of the most common misconceptions when coding for a global audience is the infamous statement: "I�ll NEVER need to do this." The most common rationalization for this misconception is that in the U.S., we only speak English, however, one in five U.S. residents speaks a foreign language at home. This is a record 61.8 million residents who speak languages other than English. Of these residents, 25.1 million (41 percent) told the Census Bureau that they speak English less than very well. Additionally, data from 2013 shows that there are seven languages with at least one million speakers in the US. These languages were Spanish (38.4 million), Chinese (three million), Tagalog (1.6 million), Vietnamese (1.4 million), French (1.3 million), and Korean and Arabic (1.1 million each). By only serving one audience, we are ignoring many customers and consumers of our solutions.  Even in my adopted hometown of Kansas City, 6% of the population speaks Spanish as their primary language. 
```

Comment:
Looks like we may have some different uses of single-quotes, which many times will cause issues on formatting.

```
"I’ll NEVER need to
```

vs.

```
"I'll NEVER need to
```